graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLResponseBody.java

```diff
@@ -51,7 +51,7 @@ public GraphQLResponseBody(Map<String, Object> specification) {
      * @return the GraphQL response body data
      */
     @JsonAnyGetter
-    @JsonInclude
+    @JsonIgnore
```

Comment:
why and when was `@JsonInclude`added?

Reply:
According to git blame, it's been there from the very beginning: https://github.com/micronaut-projects/micronaut-graphql/commit/96589153d7f0c4493b7157abeb291c8ffc050d08#diff-5185b30fe44c67e5506bf2d1b0b8cf5794ff5433a7dbfdc26ffa79f95a294678R39